az-cvm-vtpm/src/vtpm/mod.rs

```diff
@@ -126,6 +128,24 @@ impl Quote {
         let nonce = attest.extra_data().to_vec();
         Ok(nonce)
     }
+
+    pub fn verify_pcrs(&self) -> Result<bool, QuoteError> {
```

Comment:
we might want to add a unit test for the verification

Reply:
done